.ci/podSpecs/distribution.yml

```diff
@@ -66,8 +70,8 @@ spec:
       tty: true
       resources:
         limits:
-          cpu: 1
-          memory: 512Mi
+          cpu: 4
+          memory: 4Gi
         requests:
-          cpu: 1
-          memory: 512Mi
+          cpu: 4
```

Comment:
Is this really necessary?

Reply:
Ah, for requests yeah, you're right it's not necessary.